### ISSUE TITLE
404 error. "Cannot Get /"

### ISSUE THREAD

**Issue by pmaza** [aspnet/JavaScriptServices#1587]
I created a new project with the new template 8Microsoft.DotNet.Web.Spa.ProjectTemplates::2.0.0), after that I add docker support, and all worked correctly. Then I set scss as a default style and change all styles to *.scss. After this last change, the project stop working with docker. If I execute it local or just execute the angular project, all work fine, but with docker doesn't work.

**Comment by pmaza** [comment]
Hi!

Finally I've solved the problem. The problem is that I develop in windows, and then deploy the app in a linux container. In this point node-sass has only downloaded the binary for windows, so when I deploy the app in linux and try to process the *.scss files, node-sass look for the linux binding and doesn't find it.

I've solved the problem changing the start script, adding a rebuild of node-sass.
"start": "npm rebuild node-sass && ng serve --extract-css"

I don't know if this is the proper way to solve this, but it works! :)
